dpdk/vr_dpdk_lcore.c

```diff
@@ -96,6 +96,29 @@ dpdk_lcore_rx_queue_add(unsigned lcore_id, struct vr_dpdk_rx_queue *rx_queue)
 
     /* increase the number of RX queues */
     lcore->lcore_nb_rx_queues++;
+    RTE_VERIFY(lcore->lcore_nb_rx_queues < VR_DPDK_MAX_LCORE_RX_QUEUES
+            /* we use uint64_t to store RX queue masks */
+            && lcore->lcore_nb_rx_queues < 64);
```

Comment:
Isn't the use of a bitmask of 64 for lcore_rx_queues_mask a problem? Only 64 queues can be assigned to a lcore (across all VMs)? The TX queues don't use such a bitmask, so we need something similar (linked list) for RX too?